sympy/holonomic/holonomic.py

```diff
@@ -520,7 +520,7 @@ def _singularics_to_ord(self):
             y0 = []
             for i in range(a):
                 y0.append(S(0))
-            y0 += b
+            y0 += [j * factorial(a + i) for i, j in enumerate(b)]
```

Comment:
This would probably produce floats when a is happens to be one. Is the coefficient correct also when `len(b) > 1`?